cypress/integration/dib-travel/company-settings/invoice-settings.spec.ts

```diff
@@ -1,3 +1,170 @@
 describe('Company Settings - Invoice Settings (Agent)', () => {
-  // TODO: Should be covered with tests
+  beforeEach(() => {
+    cy.loginAgent();
+    cy.visit('/company-management/invoice-settings');
+  });
+  it('should display "Invoice Settings" in the sidebar navigation', () => {
+    cy.visit('');
```

Comment:
Why do we need this empty visit? cc @Nenad85

Reply:
Because I wanted to back to the home page and after that open the "Hamburger" menu, and check the title.

But we can check that on the invoce-settings page also